.prettierrc

```diff
@@ -1,4 +1,5 @@
 {
   "trailingComma": "es5",
-  "singleQuote": true
+  "singleQuote": true,
+  "arrowParens": "always"
```

Comment:
Updated this because depending on how the prettier dependency is toggled, the arrowParens setting keeps going back and forth, creating unneeded changes on my PR's.